pkgs/development/python-modules/django-allauth-2fa/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, django
+, django-allauth
+, django-otp
+, fetchFromGitHub
+, pythonOlder
+, qrcode
+, hatchling
+}:
+
+buildPythonPackage rec {
+  pname = "django-allauth-2fa";
+  version = "0.11.1";
+  format = "pyproject";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
This does not work for me smh

Reply:
The base branch seems to be out of date.
It should be able to build by rebasing to the master branch.

Reply:
Well ok, but you can merge it it seams to work anyway.